qualtran/bloqs/chemistry/df/double_factorization.py

```diff
@@ -280,10 +279,10 @@ def build_call_graph(self, ssa: 'SympySymbolAllocator') -> 'BloqCountDictT':
             in_prep_dag: 1,  # in_prep_l^dag
             rot: 1,  # rotate into system basis  listing 4 pg 54
             # apply CCZ first then CCCZ, the cost is 1 + 2 Toffolis (step 4e, and 7)
-            Toffoli(): 1,
+            ApplyControlledZs(cvs=(1, 1), bitsize=5): 1,
```

Comment:
was this incorrect before? I see the change makes it match the comment

Reply:
The previous test was only checking the `sigma` for the call graph. I updated it to check for equivalent bloq counts, which flagged this inconsistency.